packages/block-directory/src/components/downloadable-block-info/style.scss

```diff
@@ -6,7 +6,7 @@
 	display: flex;
 	justify-content: space-between;
 	color: $dark-gray-400;
-	margin-top: $grid-size;
+	margin-top: $grid-unit-10;
```

Comment:
Ultimately, I think it's better to avoid relying on this kind of variables that refer to a "value" (10 units). Instead we should prefer more semantic variables ($default-whitespace,  $button-size, ...). 

For example if you use $button-size variable here, it would mean that this value should follow the button size, if the button size changes, this value should change while when rely on "values" like here. There's no specific meaning, it makes maintenance harder. If tomorrow I change the value of this variable, am I assured that it won't break the layout?

Just a general thought, nothing to consider specifically for this PR

Reply:
I'm actually introducing those variables to help enable a grid system that works with chunks of 8 and emphasis on 12 and 24. When used as the only units of spacing, rather than arbitrary pixel numbers, they can help add consistency to the grid. There's more in https://make.wordpress.org/design/2019/10/31/proposal-a-consistent-spacing-system-for-wordpress/, which this is directly based on.